changelog_unreleased/typescript/11721.md

```diff
@@ -17,7 +17,7 @@ import { type A } from "mod";
 
 ```
```

Comment:
Since these reflect new syntax, I think it would be better to provide examples rather than showing that there used to be a syntax error:

```ts
// Example
import { type A } from "mod";
```

Reply:
Sounds good!